adapter/internal/oasparser/envoyconf/routes_with_clusters.go

```diff
@@ -215,6 +216,38 @@ func CreateRoutesWithClusters(mgwSwagger model.MgwSwagger, upstreamCerts map[str
 			routes = append(routes, routesP...)
 		}
 		return routes, clusters, endpoints, nil
+
+	}
+
+	if mgwSwagger.EndpointType == constants.AwsLambda {
+		for _, resource := range mgwSwagger.GetResources() {
+			amznResourceName := ""
+			for i, operation := range resource.GetOperations() {
+				value := model.ResolveAmznResourceName(operation.GetVendorExtensions())
+
+				if (i != 0) && (amznResourceName != value) {
+					logger.LoggerOasparser.Errorf("ARNs in all the operations of the same resource path should have the same value")
```

Comment:
Shall we log some information like APIID, API Name, Version in the log

Reply:
Unless the else condition is hit, we should not continue this block right? Here what we do is just logging the error.

Reply:
Fixed by 9b38827